Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:8.11.4
```

Comment:
Any reason we need docker for this please? Node should be enough for all the Docusaurus setup.